google/cloud/spanner/client.h

```diff
@@ -514,19 +517,21 @@ class Client {
    * or neither, in which case a single-use transaction with default options
    * is used.
    *
+   * @par Example
```

Comment:
Q.  Why is this one `@par` when the other `@par`s were changed to `###`?

Reply:
See above, reverted all of these to `@par` with some changes on what goes on the title vs. the body.